README.md

```diff
@@ -249,6 +249,40 @@ When null is the only return type on a method, as in `returns: null`, it will co
 
 Defining `returns: null | SomethingElse` on a method will compile to `Promise<null | SomethingElse>` return type.
 
+### Binaris backend
+1. Create a new directory.
+1. Switch to new directory.
+1. Run `bn create node8 add`.
+1. Create an `interface.ts` file as shown above.
+1. Generate your server code with `concord node --client fetch --server binaris example@0.0.1 interface.ts -o .` (the client is here for the sake of the example and is not required).
+1. Implement a handler:
+
+    *`function.js`*
+    ```js
+    const { ExampleWrapper } = require('./server');
+    exports.handler = ExampleWrapper.add(async (a, b) => a + b);
+    ```
+1. Export your binaris credentials: `export BINARIS_ACCOUNT_ID=$(bn show accountId) BINARIS_API_KEY=$(bn show apiKey)`
+1. Deploy your function with `bn deploy add`.
+1. Create a test file
+
+    *`test.js`*
+    ```js
+    const { ExampleClient } = require('./client');
+
+    async function main() {
+      const url = `https://run.binaris.com/v2/run/${process.env.BINARIS_ACCOUNT_ID}`;
+      const client = new ExampleClient(url, {
+        headers: {
+          'X-Binaris-Api-Key': process.env.BINARIS_API_KEY,
+        },
+      });
+      console.log('1 + 2 =', await client.add(1, 2));
```

Comment:
I suggest `try-catch` here for nicer display of possible errors (like forgot to export keys).